INFO 2023-07-18 22:05:45,302 - Start time: 20230718T220545
INFO 2023-07-18 22:05:45,423 - Trial directory made: /home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/OptOutput/US_LA2_Trans_20230718T220545/000000
INFO 2023-07-18 22:05:58,034 - Trial directory made: /home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/OptOutput/US_LA2_Trans_20230718T220545/000001
INFO 2023-07-18 22:06:12,673 - Trial directory made: /home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/OptOutput/US_LA2_Trans_20230718T220545/000002
ERROR 2023-07-18 22:08:06,826 - Boa Wrapper encountered Exception: ValueError('could not broadcast input array from shape (0,1) into shape (24,1)') in fetch_trial_data
Traceback (most recent call last):
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 438, in fetch_trial_data
    y_pred, y_true = get_model_obs(
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 195, in get_model_obs
    modelVar[i:i+24] = CurDayVar
ValueError: could not broadcast input array from shape (0,1) into shape (24,1)
ERROR 2023-07-18 22:08:06,829 - Boa Wrapper encountered Exception: ValueError('could not broadcast input array from shape (0,1) into shape (24,1)') in _fetch_trial_data
Traceback (most recent call last):
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/base_wrapper.py", line 356, in _fetch_trial_data
    res = self.fetch_trial_data(
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 438, in fetch_trial_data
    y_pred, y_true = get_model_obs(
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 195, in get_model_obs
    modelVar[i:i+24] = CurDayVar
ValueError: could not broadcast input array from shape (0,1) into shape (24,1)
ERROR 2023-07-18 22:08:06,830 - Boa Wrapper encountered Exception: ValueError('could not broadcast input array from shape (0,1) into shape (24,1)') in _fetch_trial_data
Traceback (most recent call last):
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/base_wrapper.py", line 356, in _fetch_trial_data
    res = self.fetch_trial_data(
  File "/home/tyr/BOA/boa/metaclasses.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/BOA/boa/wrappers/wrapper_utils.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 438, in fetch_trial_data
    y_pred, y_true = get_model_obs(
  File "/home/tyr/E3SM_code/components/elm/tools/OLMT/Cases/US_LA2_Trans/elm_wrapper.py", line 195, in get_model_obs
    modelVar[i:i+24] = CurDayVar
ValueError: could not broadcast input array from shape (0,1) into shape (24,1)
